gdal/frmts/jpegxr/GNUmakefile

```diff
@@ -0,0 +1,13 @@
+
+OBJ	=	jpegxrdataset.o
```

Comment:
Tabs will stay as they are consistent with other GDAL `GNUmakefile` files.

Leaving the task for bulk-untabifying of all GDAL makefiles in future.